src/controllers/product.ts

```diff
@@ -26,7 +33,10 @@ export const getPopular: RequestHandler = async (req, res) => {
   }
 };
 
-export const getNewArrivals: RequestHandler = async (req, res) => {
+export const getNewArrivals: RequestHandler = async (
+  req: Request,
+  res: Response
+) => {
   req.query.isNew = '1';
   const filter = createProductFilter(req.query);
```

Comment:
It's from another branch and pushed from here because eslint was not working when we fixed it, it discovers some errors and warnings in this file which we fix and push in this branch
so I will fix it in a new branch
Thanks